docs/developer-guide/ksqldb-clients/java-client.md

```diff
@@ -851,6 +851,30 @@ assert response.status() == 200;
 The `send()` method adds authentication headers as specified in 
 [ClientOptions](api/io/confluent/ksql/api/client/ClientOptions.html).
 
+Assert the existence of a topic or schema<a name="direct-http-requests"></a>
```

Comment:
I'm not great with my Markdown syntax... is the `<a name="direct-http-requests"></a>` stray?

Reply:
its a bad copy paste job. fixed.